azure/windows-agent/main.tf

```diff
@@ -0,0 +1,103 @@
+provider "azurerm" {}
+
+# Used to determine your public IP for forwarding rules
+data "http" "whatismyip" {
+  url = "http://whatismyip.akamai.com/"
+}
+
+locals {
+  cluster_name        = "demo52"
+  location            = "northeurope"
+  dcos_version        = "1.13.3"
+  dcos_variant        = "open"
+  dcos_instance_os    = "centos_7.6"
+  dcos_winagent_os    = "windows_1809"
+  vm_size             = "Standard_D2s_v3"
+  ssh_public_key_file = "~/.ssh/aws-meso-wind.pub"
+}
+
+module "dcos" {
+  source  = "dcos-terraform/dcos/azurerm"
+  version = "~> 0.2.0"
+
+  providers = {
+    azurerm = "azurerm"
+  }
+
+  location = "${local.location}"
+
+  cluster_name        = "${local.cluster_name}"
+  ssh_public_key_file = "${local.ssh_public_key_file}"
+  admin_ips           = ["${data.http.whatismyip.body}/32"]
+
+  num_masters        = "3"
+  num_private_agents = "1"
+  num_public_agents  = "1"
+
+  bootstrap_vm_size      = "Standard_B2ms"
+  masters_vm_size        = "${local.vm_size}"
+  private_agents_vm_size = "${local.vm_size}"
+  public_agents_vm_size  = "${local.vm_size}"
+
+  dcos_instance_os = "${local.dcos_instance_os}"
+
+  dcos_variant = "${local.dcos_variant}"
+  dcos_version = "1.13.3"
+
+  # dcos_license_key_contents = "${file("~/license.txt")}"
+  #ansible_bundled_container = "mesosphere/dcos-ansible-bundle:feature-windows-support-039d79d"
```

Comment:
should be changed to the latest one dcos-ansible release, right?

Reply:
updated